firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigGetParameterHandler.java

```diff
@@ -325,6 +347,31 @@ public FirebaseRemoteConfigValue getValue(String key) {
     return allConfigs;
   }
 
+  /**
+   * Adds a listener that will be called whenever one of the get methods is called.
+   *
+   * @param listener function that takes in the parameter value and Personalization metadata
+   */
+  public void addListener(BiConsumer<String, JSONObject> listener) {
+    synchronized (listeners) {
+      listeners.add(listener);
+    }
+  }
+
+  /**
+   * Calls all listeners in {@link #listeners}.
+   *
+   * @param value parameter value that was retrieved
```

Comment:
Wonder if there's a more precise word than retrieved here? I think it refers to the value being retrieved by the developer via a `get()` call, right?

Reply:
Reworded.